python/py-rich/Portfile

```diff
@@ -5,6 +5,7 @@ PortGroup           python 1.0
 
 name                py-rich
 version             10.1.0
+revision            1
```

Comment:
the `revision` should not be increased when adding Python versions (but ideally it should happen in a separate commit)